README.md

```diff
@@ -226,6 +222,13 @@ Specify the criteria that you recommend to be used to evaluate the code signatur
 See more info from https://developer.apple.com/library/mac/documentation/Security/Conceptual/CodeSigningGuide/RequirementLang/RequirementLang.html
 Default to `undefined`.
 
+`restrict` - *Boolean*
+
+*** To be deprecated, see `signature-flags` *** Restrict dyld loading. See doc about this [code signature flag](https://developer.apple.com/documentation/security/seccodesignatureflags/kseccodesignaturerestrict?language=objc) for more details. Disabled by default.
```

Comment:
Nit on formatting: I'm wondering if it may be better to bold the deprecation warning?

```
**To be deprecated, see `signature-flags`.**
Restrict dyld loading. See [code signature flag](https://developer.apple.com/documentation/security/seccodesignatureflags/kseccodesignaturerestrict?language=objc) for more details. Disabled by default.
```